src/Tizen.NUI/src/public/BaseComponents/ViewPublicMethods.cs

```diff
@@ -446,6 +465,36 @@ public View FindChildByName(string viewName)
             return ret;
         }
 
+        /// <summary>
+        /// Search through this view's hierarchy for a view as Animatable with the given name.
+        /// The view itself is also considered in the search.
+        /// </summary>
+        /// <pre>The view has been initialized.</pre>
+        /// <param name="childName">The name of the Animatable to find.</param>
+        /// <returns>A handle to the view as Animatable if found, or an empty handle if not.</returns>
+        /// This will not be public opened.
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public Animatable FindChildAnimatableByName(string childName)
+        {
+            //to fix memory leak issue, match the handle count with native side.
+            IntPtr cPtr = Interop.Actor.FindChildByName(SwigCPtr, childName);
+            Animatable ret = Registry.GetManagedBaseHandleFromNativePtr(cPtr) as Animatable;
+            if(ret == null)
+            {
+                // Register new camera into Registry.
+                ret = new Animatable(cPtr, true);
```

Comment:
Could you remove this warning message?
`Call System.IDisposable.Dispose on object created by 'new Animatable(cPtr, true)' before all references to it are out of scope.`

You can fix it to use `using` statement.
```suggestion
                using (ret = new Animatable(cPtr, true))
```
or to use `try` and `finally` statement.

Reply:
We need to return the ret.
If we use using we can expect the ret cannot be used outside of this method.

Reply:
I apologize that I was in such a hurry that I wrote the wrong changes before I entered the meeting!
I mean you can change this code like this :
```suggestion
                using (var animatable = new Animatable(cPtr, true))
                {
                    ret = animatable;
                }
```

Reply:
It didn't works......

Unhandled exception. System.ArgumentNullException: Value cannot be null. (Parameter 'target')
   at Tizen.NUI.Animation.AnimateTo(Animatable target, String property, Object destinationValue, AlphaFunction alphaFunction) in /home/seungho/Shared/NUI/TizenFX/src/Tizen.NUI/src/public/Animation/Animation.cs:line 784

Target handle is missing.

Reply:
As @bshsqa said, ret should be returned, it seems that "using" cannot be used in this case.
Because outside of "using" block, animatable is disposed and ret is assigned to the disposed handle..

I think that try catch can resolve this case but I am not sure if try catch is the right way because we need to apply try catch every case similar to this case..

Reply:
So, here try catch statement is the best way, but it may affect the future cases. :(
We need to think about this warning message after this commit is merged.